ui/src/helpers/get-defaults.ts

```diff
@@ -0,0 +1,159 @@
+/*
```

Comment:
> When secrets are used for setting default values, the precendence is as
> follows: organization defaults > product defaults > repository defaults.

Typo in "precendence". Also, as discussed internally, the precedence should be inverted so that default values that are "closer" to the "leaf" of the hierarchy, i.e. repository secrets, should have a higher priority that "root" default values.